Automattic/jetpack
# Protect: Add Scan tab threat count

## Descriptions
Conditionally adds a count of active threats beside the Scan tab, when threats exist.

### Other information:


## Jetpack product discussion


- https://github.com/Automattic/jetpack-scan-team/issues/1514

## Does this pull request change what data or activity we track or use?


- No

## Testing instructions:






* Use the Jetpack Beta Tester using this branch on Protect on a site with existing threats and a Scan upgrade
* Verify that the count is found next to the Scan tab
* Remove all threats and ensure a count is no longer present
* Downgrade and ensure the count logic remains

## Screenshots
![Screen Shot 2024-11-05 at 10 46 12](https://github.com/user-attachments/assets/3e2956fb-fda7-4114-9cdd-b1112872d524)
